modules/flox.nix

```diff
@@ -29,7 +29,7 @@ in
     };
   };
 
-  config = {
+  config = lib.mkIf cfg.enable {
```

Comment:
This is explicitly what caused us problems before - adding the module should include the settings, and then it's a separate setting to enable flox itself. Let's discuss